src/annotator/integrations/html.ts

```diff
@@ -65,7 +82,14 @@ export class HTMLIntegration extends TinyEmitter implements Integration {
     this._htmlMeta = new HTMLMetadata();
     this._prevURI = this._htmlMeta.uri();
 
-    this._sideBySideEnabled = this.features.flagEnabled('html_side_by_side');
+    this._sideBySideFlagEnabled =
+      this.features.flagEnabled('html_side_by_side');
+    this._sideBySideOptions = {
+      mode: initialSideBySideMode(
+        this._sideBySideFlagEnabled,
+        sideBySideOptions?.mode
+      ),
+    };
```

Comment:
If the flag is disabled when the client starts, `this._sideBySideOptions.mode` will be initialized to `manual`. If the flag later changes, the value will remain `manual` and side-by-side won't apply.

Unless the feature flag is enabled for everyone, it is turned off when `HTMLIntegration` is first instantiated and then later enabled once the client receives profile data from the API. Locally I have the feature turned on for my user, but off for everyone. The end result is that the feature remains disabled after I log in.

I think the solution can be as simple as:

```js
    this._sideBySideFlagEnabled = this.features.flagEnabled('html_side_by_side');
    this._sideBySideOptions = sideBySideOptions ?? { mode: 'auto' };
    this._sideBySideActive = false;
    this._lastLayout = null;
```

Reply:
Actually, I had to change an existing test precisely because of this. That was a smell that there was a regression. Good catch.

Reply:
> However, I think this._sideBySideOptions = sideBySideOptions ?? { mode: 'auto' }; is not enough, because if the feature flag is off and stays off but the app provided { mode: 'auto' } we are keeping the auto.

But in `fitSideBySide` you're computing the effective value as `this._sideBySideFlagEnabled && this._sideBySideOptions.mode === 'auto'`. So if the flag is off, the "auto" value will be ignored.

Reply:
> But in fitSideBySide you're computing the effective value as this._sideBySideFlagEnabled && this._sideBySideOptions.mode === 'auto'. So if the flag is off, the "auto" value will be ignored.

Good point.

Reply:
I have fixed using your suggestion. It does work indeed and I was able to revert the changes on the test I mentioned in the first comment.